src/storage-controller/src/lib.rs

```diff
@@ -29,19 +29,14 @@ use itertools::Itertools;
 use mz_build_info::BuildInfo;
 use mz_cluster_client::client::ClusterReplicaLocation;
 use mz_cluster_client::ReplicaId;
-
 use mz_ore::collections::CollectionExt;
-use mz_persist_client::stats::{SnapshotPartsStats, SnapshotStats};
```

Comment:
Random imports re-ordering, but fine for me... 😅

Reply:
I originally did have changes in this file, but they were eventually pulled into a different PR. These imports were all out of order and my IDE will automatically order them if I change the file.